### DISCUSSION TITLE
Drop JDK 11 support now?

### DISCUSSION THREAD

**Discussion by odenix** [apple/pkl#260] [General]
Supporting JDK 11 in my PRs has been quite a headache. If it will be dropped for 0.26, can we drop it right now?

**Comment by bioball** [comment]
We aren't committing to dropping Java 11 yet. We're currently figuring out whether this is doable or not.

**Reply by alexandre-frey** [reply]
The [documentation](https://pkl-lang.org/main/current/pkl-cli/index.html) states that CLI is "tested with Java 8/11/14 on macOS and Oracle Linux".
But all the JAR files seem to be with format 55.0, which is Java 11.
Could you clarify if Java 8 is supported ?

**Reply by bioball** [reply]
Ah, that is outdated! Java 8 is not supported.

**Reply by bioball** [reply]
Documentation fix in https://github.com/apple/pkl/pull/283

**Comment by odenix** [comment]
>  We're currently figuring out whether this is doable or not.

Fingers crossed. Java 17 will be a great improvement for Pkl developers and contributors.

**Comment by bioball** [comment]
Update: It's being dropped! For details, follow this PR: https://github.com/apple/pkl/pull/439
